test/controllers/graphql_controller_v2_test.rb

```diff
@@ -367,8 +369,8 @@ def setup
     refute_equal(eastern_hits, western_hits)
   end
 
-  test 'graphqlv2 geobox search with keyword search' do
-    VCR.use_cassette('graphqlv2 geobox with keyword') do
+  test 'graphql geobox search with keyword search' do
```

Comment:
Same question for this test as the other geospatial tests - should we be adding the source parameter here?

Reply:
We shouldn't need to in this case because our assertions don't care if there are more than 10,000 results.